### PR TITLE
Add edit features for trips

### PR DESCRIPTION
**Pull request by scara82**
This PR:
- Add edit feature for trip with:
     - Edit, update and helper methods in TripsController;
     - Add update and edit routes;
     - Add 'edit' button on the trips list;
    - Updates the translations.

Trello Card: https://trello.com/c/Sr5ldPkQ

### CODE REVIEW

File: app/controllers/trips_controller.rb [hunk 1]
```diff
@@ -13,9 +13,23 @@ def create
...
+    trip.update_attributes(trip_params)
+
+    respond_with @trip, location: -> { trips_path }
+  end
+
   private
 
   def trip_params
     params.require(:trip).permit!
   end
```
Comment (apalski) [inline app/controllers/trips_controller.rb:29]: Elle wasn't encouraging us to use .permit! last week?

**Comment by apalski** [comment]
LGTM!

File: app/controllers/trips_controller.rb (hunk 1 above)
Comment (scara82) [reply app/controllers/trips_controller.rb:29]: But she suggested to use it here..so I suppose it is one of this case when it is allowed ;)

**Review by elle — APPROVED** [review]

**Comment by scara82** [comment]
Thanks @elle  and @apalski , I'm merging now!

**Pull request merged**